helpers/artefact_helper.rb

```diff
@@ -0,0 +1,101 @@
+require 'sinatra/base'
+
+module Sinatra
+  module Helpers
+    module ArtefactHelper
+
+      def get_ontology_and_latest_submission
```

Comment:
this function already exist

Reply:
update the one in `application_helper.rb` to have a custom error message as argument.

Reply:
there are 4 error messages, i used just mix error message like this one
`error(404, "Ontology (artefact) '#{acronym}' not found.")`